docs/backends/Google.md

```diff
@@ -583,4 +583,80 @@ copy of the file on the reference disk, bypassing localization of the input.
 
 The Cromwell git repository includes a Java-based tool to facilitate the creation of manifest files called
 [CromwellRefdiskManifestCreatorApp](https://github.com/broadinstitute/cromwell/tree/develop/CromwellRefdiskManifestCreator).
-Please see the help command of that tool for more details.
\ No newline at end of file
+Please see the help command of that tool for more details.
+
+### Docker Image Cache Support
+
+To optimize job execution tme, Cromwell 55 and later support the use of Docker image caches on the PAPI v2 backend.
+Configuration looks like:
+
+```hocon
+backend {
+  ...
+  providers {
+    ...
+    PapiV2 {
+      actor-factory = "cromwell.backend.google.pipelines.v2beta.PipelinesApiLifecycleActorFactory"
+      config {
+        ...
+        docker-image-cache-manifest-file = "gs://path/to/a/docker/image/cache/manifest.json"
```

Comment:
TOL OOC why did this become single valued rather than a list?

Reply:
For reference files we needed a possibility to provide a list of manifests because we had `one manifest -> one reference disk` constraint. For docker image cache, each record in manifest corresponds to different disk, so I don't think there would be need to be able to set lists of manifests here. But I can be wrong of course

Reply:
Sorry, didn't realize that this is an addition, not a replacement.